Neos.Flow/Classes/Cli/ConsoleOutput.php

```diff
@@ -222,15 +268,25 @@ public function askAndValidate($question, $validator, $attempts = false, $defaul
      *
      * @param string|array $question The question to ask. If an array each array item is turned into one line of a multi-line question
      * @param callable $validator A PHP callback that gets a value and is expected to return the (transformed) value or throw an exception if it wasn't valid
-     * @param integer|boolean $attempts Max number of times to ask before giving up (false by default, which means infinite)
+     * @param integer|null $attempts Max number of times to ask before giving up (false by default, which means infinite)
```

Comment:
Should read "null by default", no?

Reply:
Still says `false by default`

Reply:
I think this view is not in sync, when i look at the latest commit (https://github.com/neos/flow-development-collection/pull/1195/commits/e096711f9de8ccfda6c2c8438281b139225cb722) I see the change

Reply:
Ah, that's how they mean "Show outdated" - not only the comment, but the outdated diff as well. Bummer. :)